helper/strutil/strutil.go

```diff
@@ -31,6 +31,19 @@ func StrListSubset(super, sub []string) bool {
 
 // Parses a comma separated list of strings into a slice of strings.
 // The return slice will be sorted and will not contain duplicate or
+// empty items.
+func ParseDedupAndSortStrings(input string, sep string) []string {
+	input = strings.TrimSpace(input)
```

Comment:
It looks like this came back with one of the commits.  This was renamed to `ParseDedupLowercaseAndSortStrings`.